src/test/java/com/google/sps/data/RouteTest.java

```diff
@@ -0,0 +1,57 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.data;
+
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+@RunWith(JUnit4.class)
+public final class RouteTest {
+
+  private static Route ROUTE_0 = new Route(0L, "Route0", true, 9L, 30L);
```

Comment:
nit: The naming suggests that this is a constant, but it is updated in the tests. In general, shared non-constant state is not a good idea for tests as it could lead to unexpected outcomes when running subsets of tests. It would be better to create a static function that creates and returns this default route and call the function from each test.

Reply:
Done